src/SDL/utility.c

```diff
@@ -3510,24 +3510,24 @@ void utyGrowthHeapFree(void *heap)
 char *utyMissingCDMessages[] =
 {
     "Invalid or missing Homeworld CD. Please insert valid CD.",
-    "CD Homeworld non valide. Veuillez ins�rer un CD valide.",
```

Comment:
You somehow broke the encoding : Accent show up nicely before, and appear as an � after.

Reply:
Original encoding seems to be Windows 1252